docs/examples/stream/_density_uncertainty.json

```diff
@@ -0,0 +1,82 @@
+[
+  {
+    "class" : "StreamDensityBasedAL",
+    "package" : "stream",
+    "method" : "StreamDensityBasedAL Sampling",
+    "category" : "Density Sampling",
+    "template" : "examples/template_stream.py",
+    "tags" : ["stream",  "classification", "single-annotator"],
+    "title" : "Density Sampling with StreamDensityBasedAL",
+    "refs" : ["settles2009active"],
+    "sequence" : ["title", "text_0", "plot", "refs"],
+    "text_0" : "",
+    "import_misc" : "from skactiveml.stream import StreamDensityBasedAL",
+    "init_qs" : "StreamDensityBasedAL(budget=0.2)",
+    "query_params" : "candidates=X_cand, clf=clf",
+    "update_params" : "candidates=X_cand, queried_indices=sampled_indices"
+  },
+  {
+    "class" : "CognitiveDualQueryStrategyRan",
+    "package" : "stream",
+    "method" : "CognitiveDualQueryStrategyRan Uncertainty Sampling",
+    "category" : "Density Sampling",
+    "template" : "examples/template_stream.py",
+    "tags" : ["stream",  "classification", "single-annotator"],
+    "title" : "Density Sampling with CognitiveDualQueryStrategyRan",
+    "refs" : ["settles2009active"],
```

Comment:
Fixe Reference for all CogDQS strategies: https://ieeexplore.ieee.org/document/9492291/